### ISSUE TITLE
Preload Tabs

### ISSUE THREAD

**Issue by martinjbaker** [driftyco/ionic#7587]
Setting `preloadTabs="true"` is no longer causing a crash in b11 which is good, but while I'm seeing some performance improvement, I'm surprised to see that images are only getting loaded when the tab is manually selected, which seems not to be a true preload.

Is this how it is supposed to work?

**Comment by jgw96** [comment]
Hello! This is expected behavior, as even thought the component is compiled, the page itself that is being shown in the tab is not fully "initialized" until navigated too. With the upcoming ahead of time compile all this will be fixed and their will be no reason to use `preloadTabs="true"` anymore as all templates will be compiled ahead of time in the build process. Thanks for using Ionic!

**Comment by martinjbaker** [comment]
That's a shame. When i was developing with Ionic 1, I tried switching through the tabs on a timeout while hiding it with the splash screen but the time taken was unreliable and ended up causing more problems than it solved, so I abandoned it.

Even putting aside seeing the images load, I'm also seeing some weird layout issues when the the tab is first switched to. Is there any way that I can force these tab pages to be fully initialized on app launch to avoid this?
